MAUI/DataForm/grouping.md

```diff
@@ -0,0 +1,308 @@
+---
+layout: post
+title: Grouping in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the grouping support in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Grouping editors in .NET MAUI DataForm
+
+The .NET MAUI DataForm support to group the editors which are relevant to each other. You can expand or collapse the group by tapping the group item.
+
+Grouping can be achieved by defining [Display] attribute or by handling the `GenerateDataFormItem` event.
+
+#### Using attributes
+
+{% tabs %}
+{% highlight c# %}
+
+public class ContactsInfo
+{
+    [Display(GroupName = "Name")]
+    public string FirstName { get; set; }
+
+    [Display(GroupName = "Name")]
+    public string MiddleName { get; set; }
+
+    [Display(GroupName = "Name")]
+    public string LastName { get; set; }    
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+{% tabs %}
+{% highlight c# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName" || e.DataFormItem.FieldName == "MiddleName" || e.DataFormItem.FieldName == "LastName")
+            {
+                e.DataFormItem.GroupName = "Name";
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing editors order in the group
+
+The order of the editors in the group can be changed by using attributes. You can set the order of data form items in group by using the `Order` property along with `GroupName` property in display attribute.
+
+{% tabs %}
+{% highlight c# %}
+public class ContactInfo
+{
+    [Display(GroupName = "Name", Order = 0)]
+    public string FirstName { get; set; }
+
+    [Display(GroupName = "Name", Order = 1)]
+    public string LastName { get; set; }
+
+    [Display(GroupName = "Name", Order = 2)]
+    public string MiddleName { get; set; }
+}
+{% endhighlight %}
+{% endtabs %}
+
+The order of the `DataFormGroupItem` can also be changed by handling the `GenerateDataFormItem` event.
```

Comment:
The order of the `DataFormGroupItem` can also be changed by handling the `GenerateDataFormItem` event using `RowOrder` property of a `DataFormItem`

Reply:
changed content